streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -433,6 +485,8 @@
      * @param valueSerde        value serdes for materializing the aggregated table,
      *                          if not specified the default serdes defined in the configs will be used
      * @param name              the name of the resulted {@link KTable}
+     *
+     * @return a {@link KTable} that contains records with unmodified keys and values that represent the latest (rolling) aggregate for each key
```

Comment:
Why the wording is different for this function and the one above? I think no matter if it is windowed or not the return KTable object should have similar semantics, but only different on windowed or not.